AndroidAnnotations/androidannotations-core/androidannotations/src/main/java/org/androidannotations/holder/EReceiverHolder.java

```diff
@@ -79,7 +79,7 @@ private void setOnReceiveIntentDataScheme() {
 		onReceiveIntentDataScheme = getOnReceiveBody().decl(getClasses().STRING, "dataScheme", getDataSchemeInvocation);
 	}
 
-	public JMethod getOnReceiveMethod() {
+	public JMethod getOnReceive() {
```

Comment:
I did this one only to keep the same naming convention. Could be kept as it was before.

Reply:
I'm ok with the change, but not with breaking stuff. maybe we could deprecate the old method and add a new one with the correct name.

or we decide that we do not mind to break plugins with a minor release.... I'm not sure and honestly never used any plugin that is not from this repo.

@WonderCsabo I think I can accept breaking plugins if you are. then we also do not need to revert the stuff from that other PR with breaking changes.

Reply:
@WonderCsabo please let us know about the "breaking plugin" thing. I've seen breaking changes between minor releases, they should just be notified correctly in the changes. And the plugin makers should then always place the AA version they support.

Reply:
@smaugho i texted with @WonderCsabo in hangouts and i think we are not going to break stuff. but he currently is busy with private life so there is no final decision yet.

Reply:
Understood @dodgex , well, I'll try to make a PR on this weekend to revert some of the breaking changes. I suggest to create the branch for version 5 then, to rebase some changes there. I could then continue building my plugin over that branch..

Reply:
@dodgex , @WonderCsabo I changed the name of this method, and rebased the branch. It should be fine now, please have a look. Thanks.

Reply:
@smaugho I'm confused. did we have this as `getOnReceiveMethod` in the past and you changed it to `getOnReceive` or was it the other way?

I mean... according to the commented diff you removed the `Method` while the PR otherwise ads `Method` to fields?

Reply:
@dodgex de original name was `getOnReceiveMethod`, I removed before `Method`, but I re-add it now, so right now, there is no change at all in that.